index.html

```diff
@@ -40,5 +44,191 @@ <h4>LET’S CONNECT</h4>
+                    <li><button class="button-category">html</button></li>
+                    <li><button class="button-category">Ruby on rails</button></li>
+                    <li><button class="button-category">css</button></li>
+                    <li><button class="button-category">javaScript</button></li>
+                </ul>
+                <button class="button-project">See project</button>
+            </div>
+        
+        </article>
+
+
+    </section>
+    <section class="about-my-self main-section" id="about-my-self main-section">
+        <div class="div-description-about">
+            <h2 class="about-header">About myself</h2>
+            <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website.
+                Look through some of my work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+        </div>
+        <div class="div-connect2">
+            <h4>LET’S CONNECT</h4>
+            <div class="social-media-icons">
+                <ul>
+                    <li><a href="#"><img class="btn" src="assets/icons/twitter.png" alt="Twitter icon"></a></li>
+                    <li><a href="#"><img class="btn" src="assets/icons/Linkedin icon.png" alt="LinkedIn icon"></a></li>
+                    <li><a href="#"><img class="btn" src="assets/icons/medium.png" alt="Medium icon"></a></li>
+                    <li><a href="#"><img class="btn" src="assets/icons/github.png" alt="Github icon"></a></li>
+                    <li><a href="#"><img class="btn" src="assets/icons/angellist.png" alt="Angellist icon"></a></li>
+                </ul>
+            </div>
+            <button class="button-get-resume">Get my resume</button>
+        </div>
+        <div class="languages">
+            <ul>
+                <li class="li-categories">
+                    <div>
+                        <h4>Languages</h4>
+                        <img src="assets/icons/Expand_Arrow_16px.png" alt="">
+                    </div>
+                    <ul>
+                        <li class="languages-items-li">
+                            <div class="languages-items">
+                                <img src="assets/images/js.png" alt="">
+                                <p>JavaScript</p>
+                            </div>
+                        </li>
+                        <li class="languages-items-li">
+                            <div class="languages-items">
+                                <img src="assets/images/html.png" alt="">
+                                <p>HTML</p>
+                            </div>
+                        </li>
+                        <li class="languages-items-li">
+                            <div class="languages-items">
+                                <img src="assets/images/css.png" alt="">
+                                <p>CSS</p>
```

Comment:
- [ ] I kindly suggest you make use of the `ul` and `li` tag to display the technologies of each categories such as `html, css javascript`. :+1: